app/src/main/java/org/go/sopt/LoginActivity.kt

```diff
@@ -0,0 +1,86 @@
+package org.go.sopt
+
+import android.content.Intent
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.util.Log
+import androidx.activity.result.ActivityResultLauncher
+import androidx.activity.result.contract.ActivityResultContracts
+import com.example.seminar1.databinding.ActivityLoginBinding
+import com.google.android.material.snackbar.Snackbar
+import org.go.sopt.model.RequestSignInDto
+import org.go.sopt.model.ResponseSignInDto
+import org.go.sopt.network.SignUpService
+import retrofit2.Call
+import retrofit2.Response
+
+class LoginActivity : AppCompatActivity() {
+
+    private lateinit var binding : ActivityLoginBinding
+    private val signInService = ServicePool.signInService
+
+    private lateinit var id : String
+    private lateinit var pw :String
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        binding = ActivityLoginBinding.inflate(layoutInflater)
+        setContentView(binding.root)
+
+        //회원가입 화면으로 이동
+        binding.signUpBtn.setOnClickListener {
+            val intent = Intent(this, SignUpActivity::class.java)
+            startActivity(intent)
+        }
+        
+        binding.loginBtn.setOnClickListener {
+
+            id = binding.idEditTv.text.toString()
+            pw = binding.pwEditTv.text.toString()
+
+            completeSignIn()
+        }
```

Comment:
기능별로 함수화 해주면 좋을 것 같네요..! 저는 startActivity 부분을 한 줄로 줄이는 게 더 좋다고 생각합니다.,! 불필요한 객체 생성을 지양해봐요~
```suggestion
        binding.signUpBtn.setOnClickListener {
            startActivity(Intent(this, SignUpActivity::class.java))
        }
        
        binding.loginBtn.setOnClickListener {

            id = binding.idEditTv.text.toString()
            pw = binding.pwEditTv.text.toString()

            completeSignIn()
        }
```